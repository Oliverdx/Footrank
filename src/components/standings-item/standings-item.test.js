import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';

import StandingsItem from './standings-item';

describe('Test if the component is showing the elements', () => {

  it('Prevent the component to not render', () => {
    const component = renderer
      .create(<StandingsItem />)
      .toJSON();

    expect(component).toMatchSnapshot();
  });

  it('Render the component correctly with the data passed', () => {
    const mock = {
      position: 1,
      team: { name: 'Lorem Ipsum' },
      won: '10',
      lost: '5',
      draw: '2',
      points: '32'
    }

    const component = renderer
      .create(<StandingsItem standings={mock} />)
      .toJSON();

    expect(component).toMatchSnapshot();
  });

});
