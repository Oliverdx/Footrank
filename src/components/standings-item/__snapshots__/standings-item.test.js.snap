// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if the component is showing the elements Prevent the component to not render 1`] = `null`;

exports[`Test if the component is showing the elements Render the component correctly with the data passed 1`] = `
<div>
  <span>
    1
  </span>
  <span
    className="team-name"
  >
    Lorem Ipsum
  </span>
  <span>
    10
  </span>
  <span>
    5
  </span>
  <span>
    2
  </span>
  <span>
    32
  </span>
</div>
`;
